cluster/test/commands_on_keys_test.rb

```diff
@@ -47,7 +47,7 @@ def test_migrate
   def test_object
     redis.lpush('mylist', 'Hello World')
     assert_equal 1, redis.object('refcount', 'mylist')
-    assert_equal 'quicklist', redis.object('encoding', 'mylist')
+    assert_equal 'listpack', redis.object('encoding', 'mylist')
```

Comment:
It looks like internal changes for Redis 7.2.

Reply:
Yes, it's best to either remove that test or to allow both values.

There is no point testing Redis internal implementation.